src/appengine/server_flask.py

```diff
@@ -47,12 +57,22 @@
 from handlers.testcase_detail import testcase_variants
 from handlers.testcase_detail import update_from_trunk
 from handlers.testcase_detail import update_issue
+from libs import helpers
 from metrics import logs
 from system import environment
 
 ndb_client = ndb.Client()
 
 
+class EmptyHandler(base_handler_flask.Handler):
+  """Empty hanlder for flask migration."""
+
+  def get(self):
+    # TODO(singharshdeep): Remove routes based on this handler
+    # after flask migration.
+    raise helpers.EarlyExitException('URL not complete.', 404)
```

Comment:
Can you explain why this is needed?

Reply:
the middleware redirects on the basis of the first path before the second '/' so paths like '/corpus-backup/make-public' won't get redirected to flask but to webapp, so just to redirect these URLs to flask I am using this.
I also considered to modify the middleware, but as currently its working well and many handlers are already migrated, I didn't wanted to modify it. Also, this hack will removed in a few days.